yeonjae/programmers-0/등수매기기.js

```diff
@@ -0,0 +1,6 @@
+function solution(score) {
+  const scoreEverage = score.map((num) => (num[0] + num[1]) / 2);
+  const sortEverage = scoreEverage.sort((a, b) => b - a);
+
+  return score.map((item) => sortEverage.indexOf((item[0] + item[1]) / 2) + 1);
+}
```

Comment:
```suggestion
function solution(score) {
  const scoreEverage = score.map((num) => num[0] + num[1]);
  const sortEverage = scoreEverage.sort((a, b) => b - a);

  return score.map((item) => sortEverage.indexOf(item[0] + item[1]) + 1);
}
```
- 풀이 잘 봤습니다 👍 `/2`는 안해도 사실 크게 상관은 없습니다